js/fetch.js

```diff
@@ -0,0 +1,123 @@
+const searchBar = document.getElementById('searchbar_container');
+const profileContainer = document.querySelector('.profile_container');
+
+//Search Bar
+
+let submitBtn = document.getElementById('submit');
+let input = document.getElementById('input');
+let noResult = document.getElementById('no_result');
+
+// submit button
+
+submitBtn.addEventListener('click', () => {
+    if (input.value !== '') {
+        fetchGit(url + input.value);
+    }
+});
+
+// Enter button
+
+input.addEventListener(
+    'keydown',
+    (e) => {
+        if (!e) {
+            let e = window.event;
+        }
+        if (e.key == 'Enter') {
+            if (input.value !== '') {
+                fetchGit(url + input.value);
+            }
+        }
+    },
+    false,
+);
+
+//Profile Container
+let avatar = document.getElementById('avatar');
+let userName = document.getElementById('name');
+let user = document.getElementById('user');
+let date = document.getElementById('date');
+let months = [
+    'Jan',
+    'Feb',
+    'Mar',
+    'Apr',
+    'May',
+    'Jun',
+    'Jul',
+    'Aug',
+    'Sep',
+    'Oct',
+    'Nov',
+    'Dec',
+];
+let bio = document.getElementById('bio');
+
+// Stats
+
+let repos = document.getElementById('repos');
+let followers = document.getElementById('followers');
+let following = document.getElementById('following');
+
+// Footer
+
+let loc = document.getElementById('location');
+let page = document.getElementById('page');
+let twitter = document.getElementById('twitter');
+let company = document.getElementById('company');
+
+// URL
+const url = 'https://api.github.com/users/';
+
+// Octacat
+fetchGit(url + 'octocat');
+
+// Fetch
+
+function fetchGit(git) {
+    fetch(git)
+        .then((response) => response.json())
+        .then((data) => {
+            profileUpdate(data);
+        })
+        .catch((error) => {
+            throw error;
+        });
+}
+
+function profileUpdate(data) {
+    if (data.message !== 'Not Found') {
+        noResult.style.display = 'none';
+        function checkNull(paramOne, paramTwo) {
+            if (paramOne === '' || paramOne === null) {
+                paramTwo.style.opacity = 0.5;
+                paramTwo.previousElementSibling.style.opacity = 0.5;
+                return 'Not available';
+            } else {
+                return `${paramOne}`;
```

Comment:
```suggestion
                return `${paramOne}`;
```
da li treba ovde vratiti opacity na `1`?